### PR TITLE
Bug 1840355 - Add configuration to add a `glean_timestamp` extra to all events

### PR DESCRIPTION
**Pull request by badboy**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,9 @@
 
 [Full changelog](https://github.com/mozilla/glean/compare/v53.2.0...main)
 
+* General
+  * Experimental: Add configuration to add a `glean_timestamp` extra to all events ([#2513](https://github.com/mozilla/glean/issues/2513))
```
Comment (travis79) [inline CHANGELOG.md:6]: Experimental, as in you are planning to test this with Nimbus? If so, that's nifty!

File: glean-core/android/src/main/java/mozilla/telemetry/glean/config/Configuration.kt [hunk 2]
```diff
@@ -31,6 +31,7 @@ data class Configuration @JvmOverloads constructor(
     val httpClient: PingUploader = HttpURLConnectionUploader(),
     val dataPath: String? = null,
     val logLevel: LevelFilter? = null,
+    val enableEventTimestamps: Boolean = false,
```
Comment (travis79) [inline glean-core/android/src/main/java/mozilla/telemetry/glean/config/Configuration.kt:34]: Please add the property to the comment above, for completeness

File: glean-core/ios/Glean/Config/Configuration.swift [hunk 3]
```diff
@@ -30,12 +31,14 @@ public struct Configuration {
         channel: String? = nil,
         serverEndpoint: String? = nil,
         dataPath: String? = nil,
-        logLevel: LevelFilter? = nil
+        logLevel: LevelFilter? = nil,
+        enableEventTimestamps: Bool = false
```
Comment (travis79) [inline glean-core/ios/Glean/Config/Configuration.swift:35]: Add this to the comment above please

**Review by travis79 — APPROVED** [review]
r+wc

File: CHANGELOG.md (hunk 1 above)
Comment (badboy) [reply CHANGELOG.md:6]: Yup, that's the plan! (but also experimental as in: don't use it, we give no guarantees)

**Pull request merged**